__tests__/app.test.js

```diff
@@ -427,3 +427,38 @@ describe("GET /api/articles (topic query)", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Respond with one object from article table, and the object should include now a comment_count", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: 1,
+          topic: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(String),
+        });
+      });
+  });
+  test("404: Respond with message 'Not found' if id is valid but not exist in database", () => {
+    return request(app)
+      .get("/api/articles/999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
+  test("400: Respond with message 'Bad request' if id in URL is not valid", () => {
+    return request(app)
+      .get("/api/articles/notAValidId")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
```

Comment:
It is not necessary to check these errors again as you should already have them for a previous task - the happy path test is enough and it is also acceptable for the happy path to just focus on comment_count being there!